src/reactive_state_access.rs

```diff
@@ -1235,6 +1359,60 @@ mod test {
     fn b_reversible() -> AtomUndo<i32> {
         0
     }
+    #[test]
+    fn test_on_changes_on_reaction() {
+        let a_b_subtraction = a_b_subtraction();
+        let mut previous = 99;
+        let mut current = 99;
+        a_b_subtraction.on_change(|p, c| {
+            previous = *p;
+            current = *c;
+        });
+        assert_eq!(previous, 0); //todo : should we expect None when init ?
+        assert_eq!(current, 0);
+        a().set(1);
+        a_b_subtraction.on_change(|p, c| {
+            previous = *p;
+            current = *c;
+        });
+        assert_eq!(previous, 0); //todo : should we expect None when init ?
+        assert_eq!(current, 1);
+        a().set(2);
+        a_b_subtraction.on_change(|p, c| {
+            previous = *p;
+            current = *c;
+        });
+        assert_eq!(previous, 1); //todo : should we expect None when init ?
```

Comment:
The unit test is failing there for some reason. 
We get previoous set to 0 et current to 1